docs/changelog/93197.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 93197
+summary: Add null check for G1 related options and origin field. This change avoids NPE when starting Elasticsearch on the Azul Platform Prime JDK.
+area: Core
```

Comment:
We have well defined areas, it should say: `Infra/Settings` instead of `Core`.

Reply:
Done!